spec/controllers/locations_controller_spec.rb

```diff
@@ -78,4 +81,46 @@
       end
     end
   end
+
+  describe "POST /update_location" do
+    context "when sign_in user is super_admin" do
+      before do
+        sign_in super_admin
+      end
+
+      context "when location's organisation belongs to signed_in user organisation" do
+        it "update location and redirect to ips_path " do
+          put :update_location, params: { location_id: location.id, location: { organisation_id: user.organisations.first.id, address: location.address, postcode: location.postcode } }
```

Comment:
It would be a good idea to check if the location has actually moved after doing this.